README.md

```diff
@@ -35,6 +35,8 @@ And several on the random number generator, around which dust is built:
 * `vignette("rng_algorithms")` on the details of algorithms used to sample from some distributions
 * `vignette("rng_package")` on using the generators from other packages
 
+The C++ API is documented in a separate [set of documentation](cpp)
```

Comment:
yeah, that's not going to be possible with this setup. As I was writing a blog post though I thought of a better way, and will implement that after. It will remove sphinx and breathe entirely from this system and create a set of additional markdown files that should easily be incorporated into the usual doc structure